src/Controls/src/Core/Platform/GestureManager/GestureManager.Windows.cs

```diff
@@ -321,6 +324,9 @@ void ClearContainerEventHandlers()
 				_container.PointerExited -= OnPointerExited;
 				_container.PointerReleased -= OnPointerReleased;
 				_container.PointerCanceled -= OnPointerCanceled;
+				_container.PointerEntered -= OnPgrPointerEntered;
+				_container.PointerExited -= OnPgrPointerExited;
```

Comment:
I see exited is also already hooked up (well unhooked in this case, but that suggests it's hooked up elsewhere) earlier. Is that expected?

Reply:
If you're referring to this line
```
_container.PointerExited -= OnPointerExited;
```

then yes, it's intended we have both. OnPointerEntered and OnPointerExited are currently being used to handle pinch, pan, and swipe gestures, whereas the newly created On**Pgr**PointerEntered and On**Pgr**PointerExited are for the new PointerGestureRecognizer